numpy/core/fromnumeric.py

```diff
@@ -3506,13 +3506,13 @@ def mean(a, axis=None, dtype=None, out=None, keepdims=np._NoValue, *,
 
 
 def _std_dispatcher(a, axis=None, dtype=None, out=None, ddof=None,
-                    keepdims=None, *, where=None):
+                    keepdims=None, *, where=None, mean=None):
     return (a, where, out)
```

Comment:
In principle, one should dispatch on `mean` if given, so `return (a, where, out, mean)`

Reply:
To be honest it is not clear to me why these dispatchers are needed and I merely followed the pattern. 

By making mean a keyword input it is no longer intended to be an ouput aswell. So why should it be returned?

Reply:
The dispatcher returns all arguments that can, in principle, contain something that is an array and hence that, if not a regular `ndarray`, can allow the function to be dealt with another package (say, `dask`). Since `mean` can be an array, it should be added (most similar to `where`).

Reply:
I see, I'll make the change then.